src/Sylius/Bundle/ApiBundle/Resources/config/api_resources/Product.xml

```diff
@@ -47,6 +47,9 @@
         <itemOperations>
             <itemOperation name="get">
                 <attribute name="path">products/{id}</attribute>
+                <attribute name="openapi_context">
+                    <attribute name="summary">As an logged administrator use $code to retrieves a product resource. As an Visitor or logged Customer use $slug to retrieves a product resource.</attribute>
```

Comment:
shouldn't we say 'use $code to retrieve'? `retrieves` is used in other tense.  Same goes for $slug